modules/scheduler-impl/src/main/java/org/opencastproject/scheduler/impl/SchedulerServiceImpl.java

```diff
@@ -1081,8 +1081,21 @@ public int getEventCount() throws SchedulerException {
   public List<MediaPackage> findConflictingEvents(String captureDeviceID, Date startDate, Date endDate)
       throws SchedulerException {
     try {
-      return persistence.getEvents(captureDeviceID, startDate, endDate, Util.EVENT_MINIMUM_SEPARATION_MILLISECONDS)
-          .parallelStream().map(this::getEventMediaPackage).collect(Collectors.toList());
+      final Organization organization = securityService.getOrganization();
+      final User user = SecurityUtil.createSystemUser(systemUserName, organization);
+      List<MediaPackage> conflictingEvents = new ArrayList();
+
+      SecurityUtil.runAs(securityService, organization, user, () -> {
+        try {
+          conflictingEvents.addAll(persistence.getEvents(captureDeviceID, startDate, endDate, Util.EVENT_MINIMUM_SEPARATION_MILLISECONDS)
+            .parallelStream().map(this::getEventMediaPackage).collect(Collectors.toList()));
```

Comment:
Similar to above, this leaks the mediapackages, doesn't it?

Reply:
Seems like it would through the /recordings/conflicts* REST endpoints. Needs some more work here as I'm currently getting a 500 error on a conflict check because of a user/org issue.